code/script.js

```diff
@@ -1,25 +1,159 @@
 const chat = document.getElementById('chat')
+const sendButton = document.getElementById('send-btn')
+const nameInput = document.getElementById('name-input')
+const form = document.getElementById('name-form')
+
 
 // Global variables, if you need any, declared here
 
+let currentQuestionNumber = 1
+let saveEvent = ""  // Saves a value outside the function
+
+// Starts here
+const greeting = () => {
+  showMessage(`Hello there, Isn't it a flowery day today? What's your name?`, 'bot')
+}
+setTimeout(greeting, 500)
+
+// Set up your eventlisteners here
+sendButton.addEventListener('click', (event) => {
+  event.preventDefault();
+  currentQuestion();
+});
+
 // Functions declared here
+// DECIDES WHICH QUESTION WE ARE HANDLING
+const currentQuestion = () => {
+  if (currentQuestionNumber === 1) {
+    handleNameInput();
+  }
+  else if (currentQuestionNumber === 2) {
+    sizeQuestion();
+  }
+  else if (currentQuestionNumber === 3) {
+    resetWindow()
+  }
+  else {
+    console.log('doesntWork')
+  }
+  // Adds a number att current question
+  currentQuestionNumber++
+}
+
+const handleNameInput = () => {
+  const name = nameInput.value;
+  showMessage(name, 'user')
+  nameInput.value = '';
+  setTimeout(() => showFlowerOptions(name), 500)
+
+  // Function that changes layout to three buttons
+  const showFlowerOptions = () => {
+    showMessage(`Hi ${name} ,what kind of event do you need flowers for?`, 'bot')
+    form.innerHTML =
+      `<div class="btn-wrapper">
+      <button id="wedding-btn" name="Wedding">WEDDING</button>
+      <button id="funeral-btn" name="Funeral">FUNERAL</button>
+      <button id="romantic-btn" name="Romantic" >ROMANTIC</button>
+      </div>`
+
+    const weddingBth = document.getElementById('wedding-btn')
+    weddingBth.addEventListener('click', () => {
+      const chosenEvent = weddingBth.name;
+      saveEvent = weddingBth.name;
+      showMessage(chosenEvent, 'user')
+      currentQuestion()
+      setTimeout(() => showMessage(`So you chosen ${chosenEvent} event. What size do you need on the bouquet?`, 'bot'), 500)
+
+    })
+
+    const funeralBth = document.getElementById('funeral-btn')
+    funeralBth.addEventListener('click', () => {
+      const chosenEvent = funeralBth.name;
+      saveEvent = funeralBth.name;
+
+      showMessage(chosenEvent, 'user')
+      currentQuestion()
+      setTimeout(() => showMessage(`So you chosen ${chosenEvent} event. What size do you need on the bouquet?`, 'bot'), 500)
+    })
+    const romanticBth = document.getElementById('romantic-btn')
+    romanticBth.addEventListener('click', () => {
+      const chosenEvent = romanticBth.name;
+      saveEvent = romanticBth.name;
+      console.log("This is", romanticBth.name)
+      showMessage(chosenEvent, 'user')
+      currentQuestion()
+      setTimeout(() => showMessage(`So you chosen ${chosenEvent} event. What size do you need on the bouquet?`, 'bot'), 500)
+    })
+
+  }
+
+};
+
```

Comment:
You could skip the spaces between the bracets just to make it look more clean. :)